From 4873d390698fe5855899c49c333f7235a0bc2589 Mon Sep 17 00:00:00 2001
From: Kyle Pitzen <kpitzen@pulumi.com>
Date: Thu, 9 Mar 2023 09:47:49 -0600
Subject: [PATCH 04/29] Marks SSE Configuration as Computed for Legacy S3
 Bucket

In January, AWS enabled SSE by default for all new S3 buckets.
Since we maintain a forked version of the legacy S3 bucket resource,
this change was not properly propagated to the legacy S3 bucket, which
appears in pulumi as `aws.s3.Bucket` and is likely the default version of
the Bucket resource that users will use.
---
 internal/service/s3legacy/bucket_legacy.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/internal/service/s3legacy/bucket_legacy.go b/internal/service/s3legacy/bucket_legacy.go
index 27fe651a83..8879116578 100644
--- a/internal/service/s3legacy/bucket_legacy.go
+++ b/internal/service/s3legacy/bucket_legacy.go
@@ -571,6 +571,7 @@ func ResourceBucketLegacy() *schema.Resource {
 				Type:     schema.TypeList,
 				MaxItems: 1,
 				Optional: true,
+				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"rule": {
